acceptancetests/winazurearm.py

```diff
@@ -292,7 +292,7 @@ def delete_instance(client, name_id, resource_group=None):
         if deployment:
             log.debug('Found {} {}'.format(resource_group.name, deployment.name))
             if not client.read_only:
-                poller = rgd.delete_vm(deployment.name)
+                poller = rgd.delete_vm(deployment.name)  # noqa ?? ./winazurearm.py:295:26: F821 undefined name 'rgd'
```

Comment:
what's this?

Reply:
ah, i think I can add lint to test to ensure running `lint` with `test` together. if we do run test, it will be good.